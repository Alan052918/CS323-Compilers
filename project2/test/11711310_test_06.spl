float foo(float a, float b) {
  return a * b;
}

int foo(int a, int b) {
  return a * b;
}

int main() {
  foo(1, 2);
}
