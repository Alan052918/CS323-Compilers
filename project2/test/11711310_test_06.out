Error type 4 at Line 5: redefine function: foo
Error type 9 at Line 10: the functionâ€™s arguments mismatch the declared parameters
